src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheckTest.java

```diff
@@ -162,15 +161,23 @@ public void testArrayInitialization()
             throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(WhitespaceAroundCheck.class);
         checkConfig.addAttribute("tokens", "ARRAY_INIT");
+        //checkConfig.addAttribute("tokens", "RCURLY");
```

Comment:
@strkkk Why do I need to add `RCURLY` ? It is in the list of default tokens. I mean `checkConfig` is a reference variable to `DefaultConfiguration`. Doesn't it contain all the default tokens?